modules/blocks.js

```diff
@@ -699,53 +979,90 @@ Blocks.prototype.count = function (cb) {
 	});
 };
 
+/**
+ * Get last block with additional 'secondsAgo' and 'fresh' properties
+ *
+ * @public
+ * @method getLastBlock
+ * @return {Object} lastBlock Modified last block
+ */
 Blocks.prototype.getLastBlock = function () {
 	if (__private.lastBlock) {
 		var epoch = constants.epochTime / 1000;
 		var lastBlockTime = epoch + __private.lastBlock.timestamp;
 		var currentTime = new Date().getTime() / 1000;
 
+		//FIXME: That function modify global last block object - not good, for what we need those properties?
```

Comment:
This function modifies global last block object. 

We should find a different way, as this can produce inconsistencies.